### DISCUSSION TITLE
Swipe function

### DISCUSSION THREAD

**Discussion by DevAdeel** [flyerhq/flutter_chat_ui#186] [Ideas & Requests]
Hi there! I am using this package to develop a  chat app. I really like this package and this saved a lot of time. I was thinking Is there any way to add swipe feature to messages?

**Comment by demchenkoalex** [comment]
Hey, sorry for the late response.. We are slowly working on that - you're talking about swipe to reply right?

**Reply by DevAdeel** [reply]
Hi,
Also sorry from me for late reply.
Yes I'm talking about swipe feature but not only for reply purpose. Like we can pass our own action to swipe function (open message receipts that was what I needed as swipe feature)

**Reply by demchenkoalex** [reply]
Hmm we will see if we could add custom functions to the swipe to reply solution

**Comment by Henry-Hiles** [comment]
Any progress on this? I'd love swipe to reply!

**Reply by demchenkoalex** [reply]
did not start yet, a bit burned out for open source atm. one of the higher priority features when im back for sure
